DUI3-DX/Converters/ArcGIS/Speckle.Converters.ArcGIS3/Geometry/ISpeckleObjectToHost/MeshToHostConverter.cs

```diff
@@ -22,11 +22,20 @@ public ACG.Multipatch RawConvert(SOG.Mesh target)
     target.TriangulateMesh();
     ACG.MultipatchBuilderEx multipatchPart = new();
     ACG.Patch newPatch = multipatchPart.MakePatch(ACG.PatchType.Triangles);
-    for (int i = 0; i < target.VerticesCount; i++)
+    for (int i = 0; i < target.faces.Count; i++)
     {
+      if (i % 4 == 0)
+      {
+        continue;
+      }
```

Comment:
I know you're calling `TriangulateMesh` right above this, but this is the type of logic that is super easy to break. If someone removes that call, the results we'll get will be highly unpredictable.